src/components/EditSections/EditReadingRoomAccess/EditReadingRoomAccess.test.js

```diff
@@ -132,21 +131,18 @@ describe('EditReadingRoomAccess', () => {
 
   it('should update access', async () => {
     renderEditReadingRoomAccess(props);
-    const accessSelectField = document.querySelectorAll('[id=reading-room-access-select]')[1];
-    await act(async () => userEvent.click(accessSelectField));
-    const list = screen.getByRole('listbox');
-    await act(async () => userEvent.click(within(list).getByText('ui-users.readingRoom.notAllowed', { exact: false })));
+    const accessSelectField = screen.getAllByRole('combobox')[1];
+    fireEvent.change(accessSelectField, { target: { value: 'ALLOWED' } });
```

Comment:
I think userEvent should be used instead of fireEvent; [explaination](https://testing-library.com/docs/user-event/intro/#difference-to-fireevent)

Reply:
Its okay to use fireEvent here, as I gave some time to implement UserEvent. With UserEvent it is not triggering the onChange Event of SelectionBox.